servicedispatcher.go

```diff
@@ -26,13 +26,20 @@ type serviceDispatcher struct {
 	// by calling findOrCreateCommand.
 	callbacks map[int]serviceCallback // guarded by mu
```

Comment:
It seems that `callbacks` is unused, anymore. And `streamCallbacks` is used for both callback and streamCallback